sensor/kubernetes/certrefresh/certrepo/service_certificates_repository.go

```diff
@@ -211,8 +214,8 @@ func (r *ServiceCertificatesRepoSecrets) createSecret(ctx context.Context, caPem
 		ObjectMeta: metav1.ObjectMeta{
 			Name:            secretSpec.SecretName,
 			Namespace:       r.Namespace,
-			Labels:          utils.GetSensorKubernetesLabels(),
-			Annotations:     utils.GetSensorKubernetesLabels(),
+			Labels:          utils.TLSSecretLabels(),
+			Annotations:     utils.GetSensorKubernetesAnnotations(),
```

Comment:
This should probably be changed in a distinct PR

```suggestion
			Labels:          utils.TLSSecretLabels(),
			Annotations:     utils.GetSensorKubernetesLabels(),
```

Reply:
`Annotations: utils.GetSensorKubernetesLabels(),`

Is this a pre-existing bug here?

Reply:
I opened a new PR for this line: https://github.com/stackrox/stackrox/pull/14023
Because it looks like a bug in the cert refresh logic, and should be fixed before the 4.7 release.

Reply:
@vladbologa I don't know if it was a bug, but it didn't look right ^^

Reply:
It was definitely wrong, so I already merged the fix. Thanks for catching this @aaa5kameric! 

That code is about to get much more important in 4.7+, because of certificate auto-rotation and Cluster Registration Secrets, it's going to be used for all Secured Cluster TLS secrets, so I wanted to have it merged ASAP.